src/config/config dist.json

```diff
@@ -1,21 +1,16 @@
 {
     "DEBUG" : true,
-    "HOST": "localhost",
-    "PORT": 5000,
-    "DB_TYPE": "mysql",
     "PRODUCTION": false,
     "CLIENT_VERBOSE": 2,
+    "HOST": "localhost",
+    "PORT": 5000,
+    "DB_URL": "mongodb://host:port/?retryWrites=true&w=majority",
+    "DAYS_TO_DELETE_DOCUMENT": 7,
     "DISCORD_WEBHOOK": null,
     "SSL": false,
     "KEY_FILE_SSL": null,
     "CERT_FILE_SSL" : null,
-    "DB_CONFIG": {
-        "DB_HOST": "db",
-        "DB_USERNAME": "root",
-        "DB_PASSWORD": "root",
-        "DB_DATABASE": "codewe",
-        "DB_PORT": "3306"
-    },
+    "REDIRECT_PORT": null,
     "METRICS": false,
     "METRICS_PORT": 8000
 }
```

Comment:
⚠️ **THIS IS A POTENTIALLY BREAKING CHANGE!** ⚠️ 
--------------------------------------------------------- 
What will break here? What and how will we change these setting in the production site `config.json`?